samples/net/mqtt/doc/provisioning.rst

```diff
@@ -0,0 +1,42 @@
+.. _mqtt_sample_provisioning:
+
+Provisioning
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+
+   This section is relevant only if you build the sample with TLS enabled.
+
+The server CA for the default MQTT broker (`test.mosquitto.org`_) is provisioned to the network stack at runtime prior to establishing a connection to the server.
+The server CA (:file:`ca-cert.pem`) is located in :file:`src/modules/transport/certs/`.
+Ensure that if the server is changed, the CA needs to be updated as well.
+
+To retrieve the server CA for a different MQTT broker, use the following command:
+
+.. code-block:: console
+
+   openssl s_client -connect <hostname>:<port> -showcerts -servername <hostname>
+
+You can use any of the certificates in the returned certificate chain.
```

Comment:
Should we recommend that? I think it's more reasonable to use root CA, as it's more stable, intermediate certificates could be replaced over time.

Reply:
Good point.

How about something like this?

```
Its recommended to use the topmost (root) certificate of the returned certificate chain as its more stable. Intermediate certificates (located further down the chain) might change over time.
```

Reply:
Sounds fine